packages/lib/src/form-fields/checkbox.wc.svelte

```diff
@@ -0,0 +1,62 @@
+<svelte:options
+  customElement={{
+    tag: 'jp-checkbox',
+    shadow: 'none',
+    extend: (customElementConstructor) => {
+      return class extends customElementConstructor {
+        static formAssociated = true;
+
+        constructor() {
+          super();
+          this.attachedInternals = this.attachInternals();
+        }
+      };
+    }
+  }}
+/>
+
+<script lang="ts">
+  import { createEventDispatcher, onMount } from 'svelte';
+
+  export let id: string | null = null;
+  export let name: string | null = null;
+  export let value: string = '';
+
+  export let attachedInternals: ElementInternals;
+
+  export let options: Array<{ label: string; value: boolean }> = [];
+  export let minSelects: number = 0;
+  export let maxSelects: number | null = null;
+
+  export const getValue = () => value;
+
+  const dispatch = createEventDispatcher();
+
+  $: {
+    if (options.filter((el) => el.value).length < minSelects) {
+      attachedInternals.setValidity({ customError: true }, 'Below limit checks.');
+    } else if (options.filter((el) => el.value).length > maxSelects) {
+      attachedInternals.setValidity({ customError: true }, 'Above limit checks.');
+    } else {
+      attachedInternals.setValidity({});
+    }
+    value = JSON.stringify(options);
+    attachedInternals.setFormValue(value);
```

Comment:
Would it be possible to add the value without using `JSON.stringify`, I'm asking I have no idea?

Reply:
We are using textarea for binding value which accepts string only.

Value usually accepts string, FormData and Blob only so I could try to do it with FormData, but I've used JSON.stringify because it binds to textarea.